service/service.go

```diff
@@ -108,10 +108,13 @@ func New(config Config) (*Service, error) {
 	var clusterapiController *clusterapi.Controller
 	{
 		c := clusterapi.ControllerConfig{
-			K8sClient:        k8sClient,
-			Logger:           config.Logger,
-			PrometheusClient: prometheusClient,
-			BaseDomain:       config.Viper.GetString(config.Flag.Service.Prometheus.BaseDomain),
+			K8sClient:                 k8sClient,
+			Logger:                    config.Logger,
+			PrometheusClient:          prometheusClient,
+			BaseDomain:                config.Viper.GetString(config.Flag.Service.Prometheus.BaseDomain),
+			SupportsPersistentStorage: config.Viper.GetBool(config.Flag.Service.Prometheus.Storage.IsPersistent),
```

Comment:
Sorry, I'm a bit lost, what are these for? They don't seem to be used here.

Reply:
Just realised these probably crept-in from #76

Reply:
Yes they did :( I failed to see it